lib/wlib/WlibConfig.h

```diff
@@ -0,0 +1,62 @@
+/**
```

Comment:
This code will be compiled with GCC when we upload it on Arduino but it is good to leave this for the meantime we work on this project.